src/components/expanded-state/custom-gas/FeesGweiInput.js

```diff
@@ -38,6 +38,7 @@ const GweiStepButton = ({
     <StepButtonWrapper
       onLongPress={onLongPress}
       onLongPressEnded={onLongPressEnded}
+      onPress={onPress}
       onPressStart={onPress}
```

Comment:
I switched to onPressStart for iOS so that when you hold down the step button, it increments by one immediately and then starts repeating

with onPress it doesn’t increment at all when you hold down until it starts repeating

guessing android button press doesn’t support onPressStart, so android should use onPress and iOS should use onPressStart

Reply:
taking another look 
`it increments by one immediately and then starts repeating` hm i think that makes sense that happens now, will try to modify it

Reply:
reverting this off this PR just to merge if needed

and yes @skylarbarrera now it should be going to develop 🚀